packages/dx-react-scheduler/src/plugins/date-navigator.tsx

```diff
@@ -24,6 +25,8 @@ const navigate = (action, currentView, intervalCount) => (direction, nextDate) =
   amount: intervalCount,
   step: currentView.type,
 });
+const navigateMemo = memoize((changeCurrentDate, currentView, intervalCount) =>
+  navigate(changeCurrentDate, currentView, intervalCount));
```

Comment:
We should use only [pure functions](https://en.wikipedia.org/wiki/Pure_function) in our code.

Reply:
React.useCallback can be used in functional component, not in classes. If we want use it, needs to refactor component.

Reply:
We should not use `memoize` in the global contex